<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>FastAPI on umemak</title><link>https://umemak.github.io/blog/tags/fastapi/</link><description>Recent content in FastAPI on umemak</description><generator>Hugo -- gohugo.io</generator><language>ja-jp</language><copyright>This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</copyright><lastBuildDate>Mon, 09 Oct 2023 00:00:00 +0000</lastBuildDate><atom:link href="https://umemak.github.io/blog/tags/fastapi/index.xml" rel="self" type="application/rss+xml"/><item><title>FastAPI入門</title><link>https://umemak.github.io/blog/posts/2023/10/09_fastapi/</link><pubDate>Mon, 09 Oct 2023 00:00:00 +0000</pubDate><guid>https://umemak.github.io/blog/posts/2023/10/09_fastapi/</guid><description>FastAPI入門をやってみた。 特にひっかかることなく進めたが、最後のテストがPASSしない。 $ docker-compose run --entrypoint &amp;#34;poetry run pytest&amp;#34; demo-app [+] Building 0.0s (0/0) docker:default [+] Building 0.0s (0/0) docker:default ======================================================= test session starts ========================================================platform linux -- Python 3.9.17, pytest-7.4.2, pluggy-1.3.0 rootdir: /src plugins: anyio-3.7.1, asyncio-0.21.1 asyncio: mode=strict collected 2 items tests/test_main.py FF [100%] ============================================================= FAILURES ============================================================= _______________________________________________________ test_create_and_read _______________________________________________________ async_client = &amp;lt;async_generator object async_client at 0x7f8d3374dca0&amp;gt; @pytest.mark.asyncio async def test_create_and_read(async_client): &amp;gt; response = await async_client.post(&amp;#34;/tasks&amp;#34;, json={&amp;#34;title&amp;#34;: &amp;#34;テストタスク&amp;#34;}) E AttributeError: &amp;#39;async_generator&amp;#39; object has no attribute &amp;#39;post&amp;#39; tests/test_main.py:41: AttributeError __________________________________________________________ test_done_flag __________________________________________________________ async_client = &amp;lt;async_generator object async_client at 0x7f8d3374ddc0&amp;gt;</description></item></channel></rss>